packages/currency/src/errors.ts

```diff
@@ -0,0 +1,26 @@
+import { RequestLogicTypes } from '@requestnetwork/types';
+
+const getSymbolAndNetwork = (
+  symbolWithNetwork: string | Omit<RequestLogicTypes.ICurrency, 'type'>,
+) => {
+  if (typeof symbolWithNetwork === 'string') {
+    const [symbol, network] = symbolWithNetwork.split('-');
```

Comment:
(unrelated to the PR) This does not work well if the symbol contains a dash. I think we already handled this somewhere else.